docs/glossary/glossary.md

```diff
@@ -9,11 +9,11 @@ Here's a list of the most commonly used terms:
-| Kyma Cluster        | A Kubernetes cluster                                                                                                                                                                                                                                                                                                                                                                                                                                                                |       |
-| Function            | A simple code snippet that you can run without provisioning or managing servers. It implements the exact business logic you define. A Function is based on the Function custom resource and can be written in either Node.js or Python. A Function can perform a business logic of its own. You can also bind it to an instance of a service and configure it to be triggered whenever it receives a particular event type from the service or a call is made to the service's API. |       |
-| Microservice        |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |       |
+| Kyma Cluster        | A Kubernetes cluster with Kyma installed. A cluster is a set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.                                                                                                                                                                                                                                                                                                                                                                                                                                                                | [Kubernetes Components](https://kubernetes.io/docs/concepts/overview/components/) |
+| Function            | A simple code snippet that you can run without provisioning or managing servers. It implements the exact business logic you define. A Function is based on the Function custom resource and can be written in either Node.js or Python. A Function can perform a business logic of its own. You can also bind it to an instance of a service and configure it to be triggered whenever it receives a particular event type from the service or a call is made to the service's API. Functions are executed only if they are triggered by an event or an API call. |       |
```

Comment:
I wonder if it makes sense to add this acronym here. 
On one hand, we don't use it again in the rest of the definition, but on the other, it feels to me like it reads easier with this `CR` there, and we already make the reader familiar with this abbr., used heavily through our docs. What do you think? 
```suggestion
| Function            | A simple code snippet that you can run without provisioning or managing servers. It implements the exact business logic you define. A Function is based on the Function custom resource (CR) and can be written in either Node.js or Python. A Function can perform a business logic of its own. You can also bind it to an instance of a service and configure it to be triggered whenever it receives a particular event type from the service or a call is made to the service's API. Functions are executed only if they are triggered by an event or an API call. |       |
```

Reply:
Thinking about it, maybe the term "Custom Resource (CR)" could even have its own entry, it's a very specific meaning that we're using in Kyma :D